File: DentRec.Domain/Entities/Payment.cs
using DentRec.Domain.Enums;

namespace DentRec.Domain.Entities
{
    public class Payment : BaseEntity
    {
        public int PatientId { get; set; }
        public int PatientLogId { get; set; }

        public decimal Amount { get; set; }
        public PaymentMethod PaymentMethod { get; set; }

        public Patient? Patient { get; set; }
        public PatientLog? PatientLog { get; set; }
    }
}

File: Client/src/app/shared/models/payment.ts
export interface NewPayment {
    patientLogId: number;
    patientId: number;
    paymentMethod: string;
    amount: number;
}

export interface Payment {
    id: number;
    patientLogId: number;
    patientId: number;
    paymentMethod: string;
    amount: number;
}

